src/logic/calculate.js

```diff
@@ -2,7 +2,6 @@ import operate from './operate';
 
 const calculate = (calculatorObj, buttonName) => {
     const{total, next, operation} = calculatorObj;
```

Comment:
we also have a problem with behaviour. Here is the scenario:
```
5 + 5 = 10
then:
5 + 2 = 12
and not 7
```
instead the last total is used as first value and takes the operation `+` and number `2` for another calculation.

Reply:
### Fixed  🤓